README.md

```diff
@@ -25,7 +25,7 @@ structures:
 * [Bifunctor](#bifunctor)
 * [Profunctor](#profunctor)
```

Comment:
I think it's best to keep this. It's nice to have a fallback in case something happens with the image, different people process information differently, and it's still plaintext so it can be searched.